sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
 # Release History
 
-## 5.0.0-beta.1 (Unreleased)
+## 5.0.0-beta.1 (2020-11-10)
+- First release.
```

Comment:
NIT - Should this have the same description as the other WebJobs Changelogs?

Reply:
Good idea, but let's just use this:
```
This is the first preview of the next generation of `Microsoft.Azure.WebJobs.Extension.Storage` which has been integrated with latest Azure Storage SDK that follows the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html) which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.
```

Reply:
for reference: this package doesn't go to nuget today. it's all shared source.

Reply:
fixed.